docs/Technical-Documentation/Architecture-Decision-Record/009-git-workflow.md

```diff
@@ -69,9 +69,13 @@ The proposed workflow below provides a remedy to these issues, as well as many o
     * opening a PR from `develop` to `HHS:main`
     * updating the PR template to change `addresses` to `closes` so that issue [can be automatically closed when the Government merges](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
     * ensuring the CI/CD pipelines are green
-    * assigning `lfrohlich` or `adpennington`as Government as reviewers. They will approve and  merge to `HHS:main` which will trigger the CI/CD process to deploy code to the prod environment. 
+    * assigning `lfrohlich` or `adpennington`as Government as reviewers. They will approve and merge to `HHS:main` which will trigger the CI/CD process to deploy code to the staging environment. 
+   
+8. `adpennington` (or her back-up) will open a PR from `HHS:main` to `HHS:master` and assign a government reviewer for PR approval. Merging this PR will trigger the CI/CD process to deploy code to the production environment. 
+   
```

Comment:
note: this still isn't in final form until we fully implement the release branching strategy outlined [here](https://github.com/raft-tech/TANF-app/blob/develop/docs/Technical-Documentation/Architecture-Decision-Record/018-versioning-and-releases.md#gitflow).